modules/table/src/main/java/org/apache/ignite/internal/table/RowChunkAdapter.java

```diff
@@ -71,6 +71,11 @@
         }
     }
 
+    /** {@inheritDoc} */
+    @Override public Tuple set(String columnName, Object value) {
+        throw new UnsupportedOperationException("Tuple is immutable.");
```

Comment:
We return an immutable tuple to the user, and looks like there is no API to convert it to a mutable one. How is the user supposed to handle "get-change-set" scenario?

Instead of throwing an exception, we can copy the data into a new mutable Tuple and return it.